values.yaml

```diff
@@ -69,30 +69,30 @@ proxy:
   # A best effort is made to parse the string and errors are ignored.
   nonProxyHosts:
 
-# When Kubernetes cluster account mode enabled, it will create a service account, cluster role, and cluster role binding
-# The created service account w/ cluster role binding is applied to the agent.
-# The agent then registers its self as a deployable Kubernetes target from within Armory Cloud.
+# When Kubernetes cluster account mode is enabled, installation creates a ServiceAccount, ClusterRole, and ClusterRoleBinding.
+# The created ServiceAccount with ClusterRoleBinding is applied to the RNA.
+# The RNA then registers itself as a deployable Kubernetes target from within CD-as-a-Service.
 #
-# When Kubernetes cluster account mode is disabled, the agent only allows you to make network calls to networked resources.
-# You will have to configure Kubernetes Accounts in the Armory Cloud Console to have deployable Kubernetes targets.
+# When Kubernetes cluster account mode is disabled, the RNA only allows you to make network calls to networked resources.
+# You have to configure Kubernetes accounts in the CD-as-a-Service Console to have deployable Kubernetes targets.
```

Comment:
Justin - is L77 still true?

Reply:
No it't as of now, we removed the ability to upload kubeconfig files manually to CD-as-a-Service